tests/phpunit/tests/test-create-delete-languages.php

```diff
@@ -146,7 +146,6 @@ public function test_language_properties_in_transient() {
 			'name',
 			'slug',
 			'term_group',
-			'term_taxonomy_id',
```

Comment:
Maybe can we had more assertions to check the content of `$term_props`. What do you think?

Reply:
It has already been done in https://github.com/polylang/polylang/pull/1161
See: https://github.com/polylang/polylang/blob/3.4-dev/tests/phpunit/tests/test-create-delete-languages.php#L172-L180

Reply:
:fish: